test/exercism/user_finished_tracks_test.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../test_helper'
+require_relative '../x_helper'
+require_relative '../integration_helper'
+
+class UserFinishedTracksTest < Minitest::Test
+  include DBCleaner
+
+  def setup
+    super
+    @user = User.create
+    @f = './test/fixtures/xapi_v3_tracks.json'
+    create_exercise('animal', 'dog')
+    create_exercise('fake', 'hello-world')
+    create_exercise('fake', 'two')
+  end
+
+  def test_user_contributions
+    X::Xapi.stub(:get, [200, File.read(@f)]) do
+      actual = UserFinishedTracks.new(@user).list
+      expected = [
+        {
+          'completed_count' => "1", 'track_id' => 'animal',
+          'total_in_track' => 1, 'language' => 'Animal'
+        },
+      ]
```

Comment:
The end result shouldn't have to care about the completed count and total in the track. This assertion should just be about the fact that it got only the `animal` track back.